src/applications/static-pages/authenticated-homepage/components/App/index.js

```diff
@@ -0,0 +1,34 @@
+// Node modules.
```

Comment:
Could we just call this `...../components/App.js`? I just not a fan of files named `index.js` since having an editor tab that simply reads `index.js` isn't very helpful :)

Reply:
Yeah I am not too opinionated on this, updated! I was just following a pattern I found elsewhere in the app.